bigbluebutton-html5/public/locales/en.json

```
@@ -1123,6 +1123,7 @@
     "app.connection-status.sessionLogs": "Session Logs",
     "app.connection-status.next": "Next page",
     "app.connection-status.prev": "Previous page",
+    "app.connection-status.lastTimeActive": "Last time user has been active",
```

```suggestion
    "app.connection-status.lastTimeActive": "Time of Last Connection Message from Client",
```

The current message is unclear... I would think it was the last time the user personally did some action in the meeting..